tidal_volume.py

```diff
@@ -0,0 +1,17 @@
+import math
+
+from causal_integral_filter import CausalIntegralFilter
+
+
+class TidalVolume():
+
+    def __init__(self, initial_time):
+        self._integral_filter = CausalIntegralFilter(0, initial_time)
+
+    def append(self, flow_rate, t):
+        self._integral_filter.append(flow_rate, t)
+        if math.isclose(flow_rate, 0, abs_tol=0.01):
```

Comment:
Is `0.01` all the more epsilon you'll need? Might be worth at least declaring a constant such that future readers have a little more information about why it's a particular value.